flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobMaster.java

```diff
@@ -759,7 +759,7 @@ public void declineCheckpoint(DeclineCheckpoint decline) {
 
 				return CompletableFuture.completedFuture(Acknowledge.get());
 			} catch (Exception e) {
-				log.error("Failed to notify KvStateRegistry about registration {}.", registrationName);
+				log.error("Failed to notify KvStateRegistry about registration {}.", registrationName, e);
```

Comment:
@kl0u Are we intentionally not logging the exception?

Reply:
@kl0u ping

Reply:
I think adding the exception logging is correct. It is logged at `error` level so it is the right thing to do I suppose.